src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/paragraph/helpers.test.js

```diff
@@ -68,4 +68,22 @@ describe('processBlock', () => {
 
     expect(processBlock(input)).toEqual(expected);
   });
+
+  it('should convert plain_text markup to candy_xml when the <link> tag is present', () => {
+    const input = {
+      text:
+        'paragraph including link <link><caption>this is a caption</caption><url href="www.example/com" platform="highweb"/></link>',
```

Comment:
Could we add a second test for the link which is in most legacy MAPs:

`<link type="page"><caption>this is a caption</caption><url href="www.example.com" platform="highweb"/></link>`

Reply:
Done!